subsys/bluetooth/audio/mcc.c

```diff
@@ -1225,8 +1493,8 @@ static uint8_t discover_mcs_char_func(struct bt_conn *conn,
 		if (!bt_uuid_cmp(chrc->uuid, BT_UUID_MCS_PLAYER_NAME)) {
 			BT_DBG("Player name, UUID: %s", bt_uuid_str(chrc->uuid));
 			cur_mcs_inst->player_name_handle = chrc->value_handle;
-			sub_params = &cur_mcs_inst->player_name_sub_params;
-			sub_params->disc_params = &cur_mcs_inst->player_name_sub_disc_params;
+			/* Use discovery params pointer as subscription flag */
+			cur_mcs_inst->player_name_sub_params.disc_params = NULL;
```

Comment:
Not related to this change, but you could save 32 bytes (16 characteristics * 2bytes per handle) if you could reuse `value_handle` stored in `bt_gatt_subscribe_params`

Reply:
I am not sure I follow.
The pointer to the discovery params can not be reused, as that is used as a persistent flag.
(This is anyway outside the scope of this PR, so I would like to take this as a separate issue.)

Reply:
I think that his point is that `playback_speed_handle` is the same value as `playback_speed_sub_params.value_handle`, so we are basically duplicating that value for each characteristic we have a `struct bt_gatt_subscribe_params` for. 

I think this is outside the scope of this PR, and a general optimization we can have in LE Audio.